docs/advanced-features/sso-permissions.md

```diff
@@ -4,27 +4,29 @@
 
 ## Overview
 
-This feature allows Quilt admin to configure what roles and admin flag value are
-assigned to the user who logs in via SSO based on the user's ID token claims.
+_SSO permissions mapping_ enables administrators to specify which roles are assigned
```

Comment:
I used wording "Quilt admin" to make it clear that it's not AWS admin
(can be ignored)

Reply:
Your wording is better but I don't want to trigger a re-review.